packages/app/src/cli/services/app-logs/logs-command/render-json-logs.ts

```diff
@@ -46,7 +48,19 @@ export async function renderJsonLogs({
 
   if (appLogs) {
     appLogs.forEach((log) => {
-      outputInfo(toFormattedAppLogJson(log, parseAppLogPayload(log.payload, log.log_type), false))
+      const storeName = storeNameById.get(log.shop_id.toString())
+      if (storeName === undefined) {
+        return
+      }
```

Comment:
In theory, this should not occur since core should not be returning logs that belong to different shops, but that fact is not expressed by the type system. In the event that it does happen due to some unpredicted bug, I've opted to just hide that log, but open to other ideas.